### ISSUE TITLE
NullReference in Shard.CreateRecordIfSpace

### ISSUE THREAD

**Issue by edmacdonald** [imazen/imageflow-dotnet-server#38]
I'm seeing this exception...
```
Object reference not set to an instance of an object.
   at Imazen.HybridCache.MetaStore.Shard.<CreateRecordIfSpace>d__16.MoveNext() in C:\projects\imageflow-dotnet-server\src\Imazen.HybridCache\MetaStore\Shard.cs:line 119
```
at this line
```
var extraLogBytes = GetLogBytesOverhead(relativePath.Length + contentType.Length);
```
because `contentType` is null

**Comment by lilith** [comment]
Fixed in 0.5.1 https://github.com/imazen/imageflow-dotnet-server/releases/tag/v0.5.1

**Comment by lilith** [comment]
Thank you so much for reporting this. I can't believe I forgot to test null values for ContentType.
